index.html

```diff
@@ -0,0 +1,1008 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="reset.css">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    
+    <title>TNW</title>
+  </head>
+  <body>
+    <header class="nav-bar-holder">
+      <nav>
+        <a href="#" class="tnw-logo">
+          <img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="The Next Web">
+        </a>
```

Comment:
- [ ] Please add the yellow banner to the navbar. It's also part of it :smile:

Reply:
Thanks done